test/unit/config.js

```diff
@@ -1 +1,4 @@
-window.config = {}
+var config = window.config = {
+  appName: 'check',
```

Comment:
this will need to be fixed when we merge back one last time into `bridge-web` - no need to do it now.